samples/snippets/csharp/VS_Snippets_CLR_System/system.String.Format/cs/starting1.cs

```diff
@@ -41,18 +41,18 @@ private static void Snippet33()
        // <Snippet33>
        int[] years = { 2013, 2014, 2015 };
        int[] population = { 1025632, 1105967, 1148203 };
-       String s = String.Format("{0,6} {1,15}\n\n", "Year", "Population");
```

Comment:
a `using System.Text;` statement also should be added after the `using System;` statement.